project/Build.scala

```diff
@@ -507,4 +512,16 @@ object Build extends sbt.Build {
         )
       )
   )
+
+  lazy val pipeline = Project(
+    id = "gearpump-experiments-pipeline",
+    base = file("experiments/pipeline"),
+    settings = commonSettings ++ myAssemblySettings ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hbase" % "hbase-client" % clouderaHBaseVersion,
+          "org.apache.hbase" % "hbase-common" % clouderaHBaseVersion
```

Comment:
Ok, i didn't know if you felt that experiments was less stable than examples. I like the way @manuzhang provided an external_kafka which added a lot of functionality related to kafka. Perhaps we can do this as well with HBaseSink in the future. For now I'll see if I can move pipeline under examples/streaming/kafka_hbase_pipeline. I may not have a new PR until tomorrow AM (currently 10:30PM PST).